### ISSUE TITLE
Python 3.7 "SyntaxError" ?

### ISSUE THREAD

**Issue by Hanoosh** [blockchain-etl/ethereum-etl#35]
`export_blocks_and_transactions.py --start-block 0 --end-block 500000 --ipc-path \\\\.\\pipe\\geth.ipc --blocks-output blocks.csv --transactions-output transactions.csv`
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\ethereum-etl-master\export_blocks_and_transactions.py", line 4, in <module>
    from ethereumetl.ipc import IPCWrapper
  File "C:\Users\Administrator\Desktop\ethereum-etl-master\ethereumetl\ipc.py", line 5, in <module>
    from web3.providers.ipc import get_default_ipc_path, PersistantSocket
  File "C:\Python37\lib\site-packages\web3\__init__.py", line 8, in <module>
    from web3.main import Web3  # noqa: E402
  File "C:\Python37\lib\site-packages\web3\main.py", line 26, in <module>
    from web3.providers.eth_tester import (
  File "C:\Python37\lib\site-packages\web3\providers\__init__.py", line 8, in <module>
    from .websocket import WebsocketProvider  # noqa: F401
  File "C:\Python37\lib\site-packages\web3\providers\websocket.py", line 9, in <module>
    import websockets
  File "C:\Python37\lib\site-packages\websockets\__init__.py", line 3, in <module>
    from .client import *
  File "C:\Python37\lib\site-packages\websockets\client.py", line 20, in <module>
    from .protocol import WebSocketCommonProtocol
  File "C:\Python37\lib\site-packages\websockets\protocol.py", line 18, in <module>
    from .compatibility import asyncio_ensure_future
  File "C:\Python37\lib\site-packages\websockets\compatibility.py", line 15
    asyncio_ensure_future = asyncio.async           # Python < 3.5
                                        ^
SyntaxError: invalid syntax

`python --version`
`Python 3.7.0`

**Comment by medvedev1088** [comment]
Thanks for reporting this. I will try to run it with Python 3.7.

**Comment by RaianeQC** [comment]
I have the same problem
python --version
Python 3.7.0

**Comment by t2y** [comment]
This issue is not ethereum-etl, but websockets. And, the issue has already fixed.

* https://github.com/aaugustin/websockets/pull/323

**Comment by t2y** [comment]
The revision is here in master branch.

* https://github.com/aaugustin/websockets/commit/9f566c5d12eae1258aa6f1991fe8cf0a7d390f97

**Comment by t2y** [comment]
This fix was included in websocket-4.0.1.

```bash
$ git describe --all 9f566c5d12eae1258aa6f1991fe8cf0a7d390f97
tags/4.0.1-3-g9f566c5
```

https://github.com/aaugustin/websockets/compare/4.0.1...master
